app/models/user.rb

```diff
@@ -313,17 +314,28 @@ def hcb_code_ids_missing_receipt
     end
   end
 
-  def transactions_missing_receipt
+  def transactions_missing_receipt(since: nil)
```

Comment:
I don't think it's safe to do this, we're memoising this value but it can change based on the argument. 

I doubt this PR would directly introduce a bug but we should be avoiding this.

https://www.honeybadger.io/blog/ruby-rails-memoization/

Reply:
`transactions_missing_receipt_count` looks good on this front

Reply:
Maybe we use https://github.com/matthewrudy/memoist - looks clean